components/Auth/AuthForm.js

```diff
@@ -42,7 +57,7 @@ export default function AuthForm(props) {
     signInOptions: [
       // Leave the lines as is for the providers you want to offer your users.
       auth.GoogleAuthProvider.PROVIDER_ID,
-      auth.FacebookAuthProvider.PROVIDER_ID,
+      // auth.FacebookAuthProvider.PROVIDER_ID,
```

Comment:
@gideonred @hannahtwobulls this was code reviewed, yes?

Reply:
This was, yes, but we've clearly missed this important code change. I'm going to look through how myself and others missed this code change.

Reply:
This probably happens during resolving the merge conflict, will double-check on it afterward.